plugin/src/main/java/net/dzikoysk/funnyguilds/data/database/DatabaseGuild.java

```diff
@@ -55,9 +57,16 @@ public static Guild deserialize(ResultSet rs) {
             if (id != null && !id.isEmpty()) {
                 uuid = UUID.fromString(id);
             }
-            
-            final User owner = User.get(os);
-            
+
+            Option<User> ownerOption = UserManager.getInstance().getUser(os);
+
+            if (ownerOption.isEmpty()) {
+                FunnyGuilds.getPluginLogger().error("Cannot deserialize guild! Caused by: owner (user instance) isn't exist");
```

Comment:
Angielski troszke do poprawy

Reply:
Angielski poprawiony ale co do tego:
> bo robienie if(costamOption.isEmpty()) return wygląda średnio i ogólnie nie do końca do tego są te option.

Nie wstawiałem w każdym miejscu `.peek` bo to by źle wyglądało, jakby cały stary kod od bazy danych miałby zostać podniesiony o dwa taby. (też warto zaznaczyć, że w tamtym miejscu powinien być null check, więc jest w zastępstwie teraz isEmpty())
 Starałem się używać funkcyjności tam gdzie było to sensowne. Wciskanie tego na siłę do takiej starej części kodu raczej nie pasuje. Celem PR jest przejście ze starych metod 
 `User#get(?) : User`
 na
 `UserManager#getUser(?) : Option<User>`
 A nie dodawanie funkcyjności gdzie się da. 
 Co prawda dodałem parę funkcyjnych usprawnień w kodzie ale tylko tam, gdzie faktycznie poprawiła się jakość kodu.
 
 Ogólna jakość tej klasy nie jest wysoka, więc można było by zrobić to funkcyjnie po refaktoryzacji `DatabaseGuild.java` w późniejszym jakiś innym PR. (Tak czy siak chyba te DB będzie do wyrzucenia)